src/design-system-team.md.njk

```diff
@@ -26,6 +26,7 @@ If you want to contact the team you can
 - Laurence de Bruxelles – Developer
 - Nora Brodian – Content Designer
 - Oliver Byford – Lead Frontend Developer, Government as a Platform
+- Owen Jones - Senior Frontend Developer
```

Comment:
I *think* this might be a sightly different dash 👀 

```suggestion
- Owen Jones – Senior Frontend Developer
```

Reply:
Aha it's using `em`'s I guess? Cheers, sorted.